data-manipulation/checksum/luhn/luhn-credit-card-validation.yml

```diff
@@ -0,0 +1,37 @@
+rule:
+  meta:
+    name: luhn credit card validation
+    namespace: data-manipulation/checksum
+    author: "@_re_fox"
+    scope: function
+    examples:
+      - 1d8fd13c890060464019c0f07b928b1a:0x401920
+      - 60abaef3fda131ffa20df480cb3f8029:0x4048e0
+  features:
+    - and:
+      - basic block:
+        - and:
+          - number: 0x0
+          - number: 0x2
+          - number: 0x4
+          - number: 0x6
+          - number: 0x8
+          - number: 0x1
+          - number: 0x3
+          - number: 0x5
+          - number: 0x7
+          - number: 0x9
+        description: Digital root lookup table
```

Comment:
whoa, @Ana06 look at this use of `description` on a statement node.

@re-fox today this won't do anything, and doesn't get displayed in the output. but it does seem like a good way of capturing your intent in a block.

Reply:
The rule has quite a few constants, I tried (my best) to clarify as much as possible in the case someone has to debug it.

Reply:
yeah, i really appreciate this. some of our other rules use `# comment` but these don't actually work very well with yaml parsers. enabling `description` on logic statements makes a lot of sense. we'll have to come up with a way to display this metadata when we emit results with `-vv`.

Reply:
I have created an issue for this: https://github.com/fireeye/capa/issues/194